ewutils.py

```diff
@@ -1236,7 +1237,7 @@ def get_move_speed(user_data):
 		move_speed *= 2
 	if ewcfg.mutation_id_fastmetabolism in mutations and user_data.hunger / user_data.get_hunger_max() < 0.4:
 		move_speed *= 1.33
-
+	move_speed = 12
```

Comment:
change debug thing back. you can just !tpp @player to get around quickly in debug mode